solr/test-framework/src/java/org/apache/solr/cloud/AbstractZkTestCase.java

```diff
@@ -62,39 +51,22 @@ public static void azt_beforeClass() throws Exception {
     System.setProperty("jetty.port", "0000");
     System.setProperty(ZOOKEEPER_FORCE_SYNC, "false");
     
-    zkServer.buildZooKeeper(SOLRHOME,
+    zkServer.buildZooKeeper(SolrTestCaseJ4.TEST_PATH(),
```

Comment:
I wouldn't inline this... at worst, copy `getSolrHome` from BaseDistributesSearchTestCase and use that. The problem is just like the comment up above says - if we're calling this from solrj or from a third party that uses our test-framework (and this is technically publicly available) then TEST_PATH won't work and they'll have no way to override/inject the correct value for their installation.

Reply:
So `AbstractZkTestCase` is only used by `TestManagedResourceStorage`, which doesn't actually use the configs stored in Zookeeper, so we can go-ahead and just remove this 🙂